save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/TestSuitesDisplayer.kt

```diff
@@ -62,6 +66,15 @@ fun ChildrenBuilder.showAvaliableTestSuites(
                         +(testSuite.tags?.joinToString(", ") ?: "")
                     }
 
+                    if (displayMode.shouldDisplayVersion()) {
+                        small {
+                            asDynamic()["data-toggle"] = "tooltip"
+                            asDynamic()["data-placement"] = "bottom"
+                            title = "Hash of commit with current test suite"
+                            +testSuite.version.take(GIT_HASH_PREFIX_LENGTH)
```

Comment:
maybe fix/enhance it on preprocessor side instead of hacking a displayed version?

Reply:
I guess it is not that important
Moreover, collision happens, so let's at least be sure it works on backend